workers/tests/unittests/test_config.py

```diff
@@ -0,0 +1,102 @@
+import inspect
+import os
+import tempfile
+import unittest
+from unittest import mock
+
+from ohsome_quality_analyst.config import config
+
+
+class TestConfig(unittest.TestCase):
+    @mock.patch.dict("os.environ", {}, clear=True)
+    def test_get_config_path_empty_env(self):
+        self.assertEqual(
+            config.get_config_path(),
+            os.path.join(
+                os.path.dirname(
+                    inspect.getfile(config),
+                ),
+                "config.yaml",
+            ),
+        )
+
+    @mock.patch.dict(
+        "os.environ",
+        {"OQT_CONFIG": os.path.join(tempfile.gettempdir(), "config.yaml")},
+        clear=True,
+    )
+    def test_get_config_path_set_env(self):
+        self.assertEqual(
+            config.get_config_path(),
+            os.path.join(tempfile.gettempdir(), "config.yaml"),
+        )
+
+    def test_load_config_from_file(self):
+        path = config.get_config_path()
+        cfg = config.load_config_from_file(path)
+        self.assertIsInstance(cfg, dict)
+        self.assertTrue(cfg)
+
+    @mock.patch.dict("os.environ", {}, clear=True)
+    def test_load_config_from_env_empty(self):
+        cfg = config.load_config_from_env()
+        self.assertEqual(cfg, {})
+
+    @mock.patch.dict(
+        "os.environ",
+        {"OQT_GEOM_SIZE_LIMIT": "200", "POSTGRES_HOST": "foo"},
+        clear=True,
+    )
+    def test_load_config_from_env_set(self):
+        cfg = config.load_config_from_env()
+        self.assertDictEqual(
+            cfg,
+            {"geom_size_limit": "200", "postgres_host": "foo"},
+        )
+
+    @mock.patch.dict("os.environ", {}, clear=True)
+    def test_get_config(self):
+        cfg = config.get_config()
+        self.assertIsInstance(cfg, dict)
+        self.assertTrue(cfg)
+
+    def test_check_missing_conf_keys_compete(self):
```

Comment:
```suggestion
    def test_check_missing_conf_keys_complete(self):
```

Reply:
This test has been removed since the function to be tested has been removed.